daprdocs/content/en/dotnet-sdk-docs/_index.md

```diff
@@ -31,6 +31,13 @@ Dapr offers a variety of packages to help with the development of .NET applicati
       <a href="{{< ref dotnet-actors >}}" class="stretched-link"></a>
     </div>
   </div>
+  <div class="card">
+    <div class="card-body">
+      <h5 class="card-title"><b>Development</b></h5>
+      <p class="card-text">Learn about local development options for .NET Dapr applications</p>
+      <a href="{{< ref dotnet-development >}}" class="stretched-link"></a>
+    </div>
+  </div>
 </div>
 <br />
 <div class="card-deck">
```

Comment:
```suggestion
</div>
<br />
<div class="card-deck">
  <div class="card">
    <div class="card-body">
      <h5 class="card-title"><b>Development</b></h5>
      <p class="card-text">Learn about local development options for .NET Dapr applications</p>
      <a href="{{< ref dotnet-development >}}" class="stretched-link"></a>
    </div>
  </div>

```

Reply:
Let's put this card as the first one in the second row instead of the last card of the first